components/config/src/config/taxonomies.rs

```diff
@@ -32,18 +32,10 @@ impl Default for TaxonomyConfig {
 
 impl TaxonomyConfig {
     pub fn is_paginated(&self) -> bool {
-        if let Some(paginate_by) = self.paginate_by {
-            paginate_by > 0
-        } else {
-            false
-        }
+        self.paginate_by.is_some_and(|paginate_by| paginate_by > 0)
```

Comment:
TIL about `is_some_and`

Reply:
The method name is explicit but uncommon. It is often called `exists()` ([for example in Scala](https://www.scala-lang.org/api/2.13.3/scala/Option.html#exists(p:A=%3EBoolean):Boolean)) in other functional languages, because it can be applied to construct of more than one element, such as lists, with the same semantics.

I've also seen `self.paginate_by.into_iter().any(|paginate_by| paginate_by > 0)`  used before `is_some_and()` was introduced.